src/index.js

```diff
@@ -109,11 +113,27 @@ const hasRedirect = () => {
     return config.enableNewSignIn ? SignInController : LoginController;
 };
 
+const logoutUser = async () => {
+    try {
+        const redirect = new URLSearchParams(window.location.search).get("redirect");
+        user.logOut();
+        if (redirect) {
+            handleRedirect(redirect);
+        } else {
+            browserHistory.push(`${rootPath}/login`);
+        }
+    } catch (error) {
+        console.error("Error during logout:", error);
+        browserHistory.push(`${rootPath}/login`);
+    }
+};
+
```

Comment:
I think the logic here should be - 

User is in downstream system and hits logout
Client makes request to /logout?redirect=/myPath
User is logged out and redirected to the login screen (/florence/login?redirect=/myPath)
User logs in again and is redirected back to the downstream system (/myPath)

I think. 

But perhaps this is just overcomplicating things.